ttnn/cpp/ttnn/operations/uniform/uniform.cpp

```diff
@@ -0,0 +1,21 @@
+
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "uniform.hpp"
+
+#include "device/uniform_device_operation.hpp"
+
+namespace ttnn::operations::uniform {
+
+Tensor Uniform::invoke(
+    const Tensor &input_tensor,
+    const float from,
+    const float to,
+    const std::optional<ttnn::MemoryConfig> &memory_config,
+    std::optional<const ttnn::DeviceComputeKernelConfig> compute_kernel_config) {
```

Comment:
Please use const std::optional<DeviceComputeKernelConfig>& compute_kernel_config for consistency.

Reply:
Updated